src/pages/news/2020-06-27-анонс-tbtc-js.ru.md

```diff
@@ -0,0 +1,23 @@
+---
+template: news-item
+title: Анонс tbtc.js
+description: Анонс tbtc.js
+date: 2020-06-27T15:20:26.304Z
```

Comment:
This date is wrong- we want to keep the date of the original material, both in the path and metadata